isis/src/base/objs/ProcessExportPds4/ProcessExportPds4.cpp

```diff
@@ -165,104 +189,186 @@ namespace Isis {
    * Instrument group to the PDS4 labels. 
    */
   void ProcessExportPds4::standardInstrument() {
-    Pvl *inputLabel = InputCubes[0]->label(); 
-    FileName transfile;
-
-    // Translate the Instrument group
-    transfile = "$base/translations/pds4ExportInstrument.trn";
-    PvlToXmlTranslationManager instXlator(*inputLabel, transfile.expanded());
-    instXlator.Auto(*m_domDoc);
-
-    // If instrument and spacecraft values were translated, create the combined name
-    QDomElement obsAreaNode = m_domDoc->documentElement().firstChildElement("Observation_Area");
-
-    if ( !obsAreaNode.isNull() ) {
-
-      // fix start/stop times, if needed
-      QDomElement timeNode = obsAreaNode.firstChildElement("Time_Coordinates");
-      if (!timeNode.isNull()) {
-        QDomElement startTime = timeNode.firstChildElement("start_date_time");
-        if (startTime.text() == "") {
-          startTime.setAttribute("xsi:nil", "true");
-        }
-        else {
-          QString timeValue = startTime.text();
-          PvlToXmlTranslationManager::resetElementValue(startTime, timeValue + "Z");
-        }
-        QDomElement stopTime  = timeNode.firstChildElement("stop_date_time"); 
-        if (stopTime.text() == "") {
-          stopTime.setAttribute("xsi:nil", "true");
-        }
-        else {
-          QString timeValue = stopTime.text();
-          PvlToXmlTranslationManager::resetElementValue(stopTime, timeValue + "Z");
+    Pvl *inputLabel = InputCubes[0]->label();
+    FileName translationFileName;
+
+    if (inputLabel->findObject("IsisCube").hasGroup("Instrument")) {
+      
+      // Translate the Instrument group
+      translationFileName = "$base/translations/pds4ExportInstrument.trn";
+      PvlToXmlTranslationManager instXlator(*inputLabel, translationFileName.expanded());
+      instXlator.Auto(*m_domDoc);
+      
+      // If instrument and spacecraft values were translated, create the combined name
+      QDomElement obsAreaNode = m_domDoc->documentElement().firstChildElement("Observation_Area");
+      
+      if ( !obsAreaNode.isNull() ) {
+      
+        // fix start/stop times, if needed
+        QDomElement timeNode = obsAreaNode.firstChildElement("Time_Coordinates");
+        if (!timeNode.isNull()) {
+          QDomElement startTime = timeNode.firstChildElement("start_date_time");
+          if (startTime.text() == "") {
+            startTime.setAttribute("xsi:nil", "true");
```

Comment:
More notes: I think Jeannie mentioned that this particular line is not working as expected.